pkg/controller/replication_controller.go

```diff
@@ -72,6 +74,20 @@ func (r RealPodControl) createReplica(namespace string, controller api.Replicati
 		desiredAnnotations[k] = v
 	}
 
+	createdByRef, err := api.GetReference(&controller)
+	if err != nil {
+		util.HandleError(fmt.Errorf("unable to get controller reference: %v", err))
+		return
+	}
+
+	createdByRefJson, err := json.Marshal(createdByRef)
```

Comment:
Also, now that you got me actually thinking about it, this is gonna be a nightmare for any sort of schema updating script-- it's going to have to understand the annotations to be correct.